### DISCUSSION TITLE
Urgent Assistance Needed - GitHub Account Recovery

### DISCUSSION THREAD

**Discussion by ashish02970** [community/community#78695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this email finds you well. I am writing to seek urgent assistance with recovering access to my GitHub account, which is linked to the username hyper0923. Unfortunately, I am encountering difficulties with the two-factor authentication (2FA) process, and my recovery codes seem to be ineffective.

Here are the details of my situation:

- **Username**: hyper0923
- **Registered Email**: [pixalme@gmail.com](mailto:pixalme@gmail.com) (Please note that I have unlinked this email from my GitHub account)
- **Recovery Codes**: I possess recovery codes, but they are not working.

I am confident about my password and username, but the 2FA process is preventing me from accessing my account. I have private repositories associated with this account that are crucial for my ongoing projects.

I have attempted the following steps:
1. Used recovery codes without success.
2. Unlinked the email associated with the account, making email-based recovery unfeasible.

I kindly request your guidance and support in resolving this matter promptly. If there are additional steps or information needed to verify my identity, please let me know, and I will provide the necessary details.

I understand the importance of account security, and I appreciate your commitment to safeguarding user accounts. Your prompt attention to this matter is highly valued, and I look forward to regaining access to my GitHub account as soon as possible.

Thank you for your understanding and assistance.

Best regards,

Ashish Sharma
[pixalme@gmail.com](mailto:pixalme@gmail.com)

**Comment by queenofcorgis** [comment]
Hi @ashish02970 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by ashish02970** [comment]
I have already visited there but that is virtual assistant they does not understand my case.
